src/main/java/seedu/duke/exception/GetJackDException.java

```diff
@@ -0,0 +1,10 @@
+package seedu.duke.exception;
+
+/**
+ * Prints the current error message provided during an exception
```

Comment:
Perhaps, you can mention that this Exception is a custom made exception used throughout the app where the developer can add in their own error messages by calling the constructor like so: GetJackDException("Error Message")

Reply:
Thank you for the suggestion, will do!